structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/cdp/CDPStructuredRestCallEvent.java

```diff
@@ -0,0 +1,38 @@
+package com.sequenceiq.cloudbreak.structuredevent.event.cdp;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.cloudbreak.structuredevent.event.rest.RestCallDetails;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CDPStructuredRestCallEvent extends CDPStructuredEvent {
+    private RestCallDetails restCall;
+
+    public CDPStructuredRestCallEvent() {
+        super(CDPStructuredRestCallEvent.class.getSimpleName());
+    }
+
+    public CDPStructuredRestCallEvent(CDPOperationDetails operation, RestCallDetails restCall) {
+        super(CDPStructuredRestCallEvent.class.getSimpleName(), operation);
+        this.restCall = restCall;
+    }
+
+    @Override
+    public String getStatus() {
+        return restCall.getRestResponse().getStatusText() != null
```

Comment:
are we sure restcall has always a non null restresponse? I don't see anything in `RestCallDetails` what would ensure that

Reply:
yes, here: `CDPStructuredEventFilter#326-335`